### PR TITLE
New pages and modifications

### PR DESCRIPTION
**Pull request by Jacobjashwanth**
Added new pages for CS IT Department Policies and Procedures, Dreamspark/Microsoft Imagine, modifications in header and menu files

### CODE REVIEW

File: README.md
Comment (haehn) [inline README.md:1]: This should not be included unless you update the readme and include all students from the previous versions etc.

Comment (Jacobjashwanth) [reply README.md:1]: I don't know somehow the previous versions are including in my pull request. Unfortunately, I can't fix that issue.

Comment (Jacobjashwanth) [reply README.md:1]: I removed the readme file now

**Comment by haehn** [comment]
I merged it now thank you

**Pull request merged**